content/numerical/PolynomialBase.h

```diff
@@ -0,0 +1,57 @@
+/**
+ * Author: chilli, Andrew He, Adamant
+ * Date: 2019-04-27
+ * Description: A FFT based Polynomial class.
+ */
+#pragma once
+
+#include "../number-theory/ModularArithmetic.h"
+#include "FastFourierTransform.h"
+#include "FastFourierTransformMod.h"
+// #include "NumberTheoreticTransform.h"
+
+typedef Mod num;
+typedef vector<num> poly;
+vector<Mod> conv(vector<Mod> a, vector<Mod> b) {
+	auto res = convMod<mod>(vl(all(a)), vl(all(b)));
+	// auto res = conv(vl(all(a)), vl(all(b)));
+	return vector<Mod>(all(res));
+}
```

Comment:
Is this layer of wrapping useful? Why not put it directly in `operator *=`?

Reply:
Good point - I was originally testing out a couple different ideas to encapsulate `FFT`, and wasn't sure what'd be necessary.